### PR TITLE
AT_14.07_001 | Multi-configuration project>Delete Multi-configuration project within itself

### PR DESCRIPTION
**Pull request by AlexSQE**
https://trello.com/c/NiInsYPN/958-at1407001-multi-configuration-projectdelete-multi-configuration-project-within-itself

### CODE REVIEW

File: cypress/e2e/multiCofigurationProjectDelete.cy.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        cy.get('input#name').type(projects.multiConfigurationProject.name);
+        cy.get('.hudson_matrix_MatrixProject').click();
+        cy.get('#ok-button').click();
+        cy.get("button[name='Submit']").click();
+        cy.get("ol#breadcrumbs > li:nth-of-type(1)").click();
+    })
+
+    it('AT_14.07_001 | MCP | Delete Multi-configuration project within itself', () => {
+        cy.get(`a[href='job/${projects.multiConfigurationProject.name}/']`).click();
+        cy.get(".confirmation-link").contains("Delete Multi-configuration project").click();
```
Comment (IrynaKolh) [inline cypress/e2e/multiCofigurationProjectDelete.cy.js:16]: перенести лучше в fixtures строковую переменную

File: cypress/e2e/multiCofigurationProjectDelete.cy.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    beforeEach(() => {
+        cy.get('a[href="newJob"]').click();
+        cy.get('input#name').type(projects.multiConfigurationProject.name);
+        cy.get('.hudson_matrix_MatrixProject').click();
+        cy.get('#ok-button').click();
+        cy.get("button[name='Submit']").click();
+        cy.get("ol#breadcrumbs > li:nth-of-type(1)").click();
+    })
+
+    it('AT_14.07_001 | MCP | Delete Multi-configuration project within itself', () => {
```
Comment (IrynaKolh) [inline cypress/e2e/multiCofigurationProjectDelete.cy.js:14]: может не стоит сокращать Multi Configuration Project Delete?

**Review by IrynaKolh — COMMENTED** [review]
Good Job!

**Review by IrynaKolh — APPROVED** [review]

File: cypress/e2e/multiCofigurationProjectDelete.cy.js (hunk 2 above)
Comment (AlexSQE) [reply cypress/e2e/multiCofigurationProjectDelete.cy.js]: Agree

File: cypress/e2e/multiCofigurationProjectDelete.cy.js (hunk 1 above)
Comment (AlexSQE) [reply cypress/e2e/multiCofigurationProjectDelete.cy.js]: Done

**Review by IraKalinka — APPROVED** [review]

**Review by IraKalinka — APPROVED** [review]

**Pull request merged**